client/__tests__/e2e/cellxgeneActions.ts

```diff
@@ -769,19 +787,67 @@ export async function assertUndoRedo(
 }
 
 export async function snapshotTestGraph(page: Page, testInfo: TestInfo) {
-  const buttonID = "capture-and-display-graph";
   const imageID = "graph-image";
 
+  await waitUntilNoSkeletonDetected(page);
+
   await tryUntil(
     async () => {
-      await page.getByTestId(buttonID).click({ force: true });
+      await page.getByTestId(GRAPH_AS_IMAGE_TEST_ID).click({ force: true });
 
       await page.getByTestId(imageID).waitFor();
 
       await takeSnapshot(page, testInfo);
+
+      /**
+       * (thuang): Remove the image in the DOM after taking the snapshot
+       */
+      await page.getByTestId(GRAPH_AS_IMAGE_TEST_ID).click({ force: true });
```

Comment:
this is needed to ensure the graph canvas is visible if we perform further actions that update the canvas